### PR TITLE
[bitnami/grafana-operator] Add functionality to pass the env list as implemented in CRD

### PR DESCRIPTION
**Pull request by AlexxxxxxD**
### Description of the change

Introduced the capability to pass custom environment variables to the Grafana container via `values.yaml` and made corresponding updates in `grafana.yaml` to apply these environment variables.

### Benefits

Users can now specify custom environment variables for the Grafana container. This can be useful for configuring and tweaking various aspects of Grafana, such as setting the admin credentials via Kubernetes secrets.

### Possible drawbacks

No known drawbacks or limitations for this feature at this time.

### Applicable issues

N/A

### Additional information

New parameters have been added to `values.yaml` to configure environment variables for the Grafana container, along with the logic to apply them in `grafana.yaml`.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO error?

**Comment by AlexxxxxxD** [comment]
@javsalgar, hi! 

> Could you fix the DCO error?

I've corrected it.

File: bitnami/grafana-operator/Chart.yaml [hunk 1]
```diff
@@ -30,4 +30,4 @@ maintainers:
 name: grafana-operator
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/grafana-operator
-version: 3.4.0
+version: 3.4.1
```
Comment (corico44) [inline bitnami/grafana-operator/Chart.yaml:33]: Could you increase the version in a minor instead of a patch, please? It should look like this: `3.5.0`

**Review by corico44 — APPROVED** [review]
LGTM

**Comment by corico44** [comment]
@AlexxxxxxD the branch has some conflicts. Can you resolve them, please?

**Pull request closed without merging**